sys/net/gnrc/application_layer/uhcpc/gnrc_uhcpc.c

```diff
@@ -69,58 +63,13 @@ static void set_interface_roles(void)
              "interface.\n", gnrc_border_interface, gnrc_wireless_interface);
 }
 
-static ipv6_addr_t _prefix;
-static uint8_t _prefix_len;
-
-#ifdef MODULE_GNRC_SIXLOWPAN_CTX
-#define SIXLO_CTX_LTIME_MIN (60U)   /**< context lifetime in minutes */
-
-static bool _ctx_match(const gnrc_sixlowpan_ctx_t *ctx,
-                       const ipv6_addr_t *prefix, uint8_t prefix_len)
-{
-    return (ctx != NULL) &&
-           (ctx->prefix_len == prefix_len) &&
-           (ipv6_addr_match_prefix(&ctx->prefix, prefix) >= prefix_len);
-}
-
-static void _update_6ctx(const ipv6_addr_t *prefix, uint8_t prefix_len)
-{
-    gnrc_sixlowpan_ctx_t *ctx = gnrc_sixlowpan_ctx_lookup_addr(prefix);
-    uint8_t cid = 0;
-
-    if (!_ctx_match(ctx, prefix, prefix_len)) {
-        /* While the context is a prefix match, the defined prefix within the
-         * context does not match => use new context */
-        ctx = NULL;
-    }
-    else {
-        cid = ctx->flags_id & GNRC_SIXLOWPAN_CTX_FLAGS_CID_MASK;
-    }
-    /* find first free context ID */
-    if (ctx == NULL) {
-        while (((ctx = gnrc_sixlowpan_ctx_lookup_id(cid)) != NULL) &&
-               !_ctx_match(ctx, prefix, prefix_len)) {
-            cid++;
-        }
-    }
-    if (cid < GNRC_SIXLOWPAN_CTX_SIZE) {
-        LOG_INFO("gnrc_uhcpc: uhcp_handle_prefix(): add compression context "
-                 "%u for prefix %s/%u\n", cid,
-                 ipv6_addr_to_str(addr_str, prefix, sizeof(addr_str)),
-                 prefix_len);
-        gnrc_sixlowpan_ctx_update(cid, (ipv6_addr_t *)prefix, prefix_len,
-                                  SIXLO_CTX_LTIME_MIN, true);
-    }
-}
-#endif
-
 void uhcp_handle_prefix(uint8_t *prefix, uint8_t prefix_len, uint16_t lifetime,
                         uint8_t *src, uhcp_iface_t iface)
 {
-    (void)lifetime;
+    int idx;
+    gnrc_netif_t *downstream;
```

Comment:
I would prefer to keep the terminology used throughout the code here, so

```suggestion
    gnrc_netif_t *wireless;
```